src/plugins/recurring-issues/index.js

```diff
@@ -15,9 +15,40 @@ const moment = require("moment-timezone");
 async function getReleaseIssueBody(releaseDate) {
     return `
 
-The scheduled release on ${releaseDate.format("dddd, MMMM Do, YYYY")} is assigned to:
+The next scheduled release will occur on ${releaseDate.format("dddd, MMMM Do, YYYY")}.
 
-* (needs volunteer)
+## Release Day Checklist
+
+- [ ] Remove the 'tsc agenda' label on this issue
+- [ ] Review open pull requests and merge any that are ready
+- [ ] Start the release on [Jenkings](https://jenkins.eslint.org)
```

Comment:
There were a couple lines about 2FA in the [old instructions](https://eslint.org/docs/maintainer-guide/releases), and I think the primary use for this here will be to jog memory around where one should look for the code. If we ever forget to add a new maintainer to the vault, it also tells them what to ask about if they've gotten stuck at this step without it.

```suggestion
- [ ] Start the release on [Jenkings](https://jenkins.eslint.org)
- [ ] Wait for the build to pause with a 2FA prompt and enter the six-digit code from the team's shared 1Password vault
```

Reply:
Since this will be public, do we want to display that info? I left it off initially because I wasn't sure if we wanted to advertise this to the public.

Reply:
I had a similar internal debate about the value of obscurity on this but decided all the layers of checks and approval someone would have to get through to gain access are likely sufficient. I don't feel strongly about it, however.

Reply:
I think the prompt is fairly self-explanatory (it provides a link to type in the OTP), so even if we don't include this it shouldn't be too much of a burden for someone to ask the rest of the team what they're missing.